src/contexts/videoContext.js

```diff
@@ -162,13 +183,52 @@ export const VideoProvider = ({ children }) => {
 		}
 	};
 
+	const addToHistoryVideos = async (axiosRequest, video) => {
+		try {
+			const historyVideosURL = "/api/user/history";
+			const { output } = await axiosRequest({
+				method: "POST",
+				url: historyVideosURL,
+				resKey: "history",
+				alert: "Added to history",
+				data: { video: video },
+			});
+			videoDispatch({
+				type: "POST_HISTORY_VIDEOS",
+				payload: output,
+			});
+		} catch (error) {
+			console.log("from addToHistoryVideos", error);
```

Comment:
Please try using alert or toast display instead of console log

Reply:
I don't want to show this error to the user, its for only development purpose. will remove it in final draft